### ISSUE TITLE
[backup] Possibility to snooze alarms during workflow duration

### ISSUE THREAD

**Issue by Klaitos** [terraform-google-modules/terraform-google-sql-db#428]
### TL;DR

Hello, 
I've made this update on my fork : I can snooze some alarms during the workflow duration. (for example I have a CPU alarm ringing during the sql export but it is an expected ring)
I do not know if this something you want to be included in this module or it is out of his scope so let me know and I can open a PR for you. (with some cleaning/adjustments)

https://github.com/appchoose/terraform-google-sql-db/pull/1/files

Example : 

<img width="961" alt="Capture d’écran 2023-02-28 à 10 41 40" src="https://user-images.githubusercontent.com/644360/221814405-390c5a3f-9c49-4abd-8ce5-6c9b0ae173b1.png">

Thanks
